Backend/Database/Database.cs

```diff
@@ -16,7 +16,7 @@ public static void SetUp()
         {
             TextMessageContext messageContext = new TextMessageContext();
             // This vvv should be disabled in prod
-            messageContext.Database.EnsureDeleted();
+            // messageContext.Database.EnsureDeleted();
```

Comment:
Why purpose does this serve?

Reply:
It clears the database from messages when starting the application, useful for testing